src/daemon/src/df-daemon/initializer/initializer.go

```diff
@@ -69,7 +71,7 @@ func cleanLocalRepo() {
 						log.Warnf("walk file:%s error:%v", path, err)
 					} else {
 						if info.Mode().IsRegular() {
-							if time.Now().Unix() - reflect.ValueOf(info.Sys()).Elem().FieldByName("Atim").Field(0).Int() >= 3600 {
+							if time.Now().Unix()-reflect.ValueOf(info.Sys()).Elem().FieldByName("Atim").Field(0).Int() >= 3600 {
```

Comment:
I think this block of code are not so good. I recommend that we should use more gotools to guarantee the code format, lint, vet and some more other things.

Reply:
There are more bits needs to be revisited, which I would like to do several PRs as a follow up, and make those changes gradually.

The tooling part which I believe we can track it as an issue.

Reply:
ok,you can do it,I will review your pull request as soon as possible.